libtenzir/include/tenzir/pcapng.hpp

```diff
@@ -0,0 +1,22 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include <cstdint>
+
+/// PCAPng utilities and data structures as defined in the IETF draft at
+/// https://datatracker.ietf.org/doc/draft-tuexen-opsawg-pcapng/. Visit
+/// https://pcapng.com/ for a high-level overview about the PCAPng format.
+namespace tenzir::pcapng {
```

Comment:
A separate header + namespace is maybe a tiny little bit of overkill? ;)

This probably could have just been `pcapng_magic_number` in `pcap.hpp`, but on the other hand having one more header doesn't hurt anyone so feel free to merge as-is.

Reply:
I hear you. The reason why I did this is because we're going to populate that header as we add PCAPng incrementally. Overkill for what we have hear, but it sets the framing.